pkgs/servers/misc/airsonic-advanced/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchurl, nixosTests }:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```
we don't compile anything.